en/snap/integration-guide.md

```diff
@@ -411,55 +437,103 @@ Or try the demo here:
   " class="my-btn">Try Snap Demo &#9099;</button>
 </p>
 
-After payment completed, customer will be redirected back to `Finish URL` [specified on Midtrans Dashboard](/en/snap/advanced-feature.md#configure-redirection-url), under menu **Settings > Snap Preference > System Settings > `Finish URL`**.
+
+After the payment is completed, customer is redirected back to `Finish URL`. It is specified on [Midtrans Dashboard](/en/snap/advanced-feature.md#configure-redirection-url), under menu **Settings > Snap Preference > System Settings > `Finish URL`**.
+
+<details>
+<summary><b>Configuring Finish Redirect URL</b></summary>
+<article>To configure the *Finish Redirect URL*, follow the steps given below.
+
+1. Login to your MAP account.
+
+2. On the Home page, go to **SETTINGS > CONFIGURATION**.
+   *Configuration* page is displayed.   
+
+3. Enter **Finish Redirect URL** with your landing page endpoint.
+
+4. Click **Update**.
+   A confirmation message is displayed.
+
+   ![Core API](/Users/rashmi/Desktop/technical-documentation-site/asset/image/coreapi/core-api-finish-redirect-url-2.png)
+
+   The *Finish Redirect URL* is configured.
```

Comment:
This is brokenly rendered on browser view:
<img width="791" alt="Screen Shot 2020-11-20 at 12 23 47" src="https://user-images.githubusercontent.com/13027142/99762572-6ac7d000-2b2b-11eb-915f-3de9c977e22d.png">

I saw `/Users/rashmi/Desktop/` file path for the image.
Did this part got copy-pasted without being tested via browser viewing?